config/initializers/assets.rb

```diff
@@ -10,3 +10,10 @@
   application-ie7.css
   application-ie8.css
 )
+
+# Precompile js
+Rails.application.config.assets.precompile += %w(
+  govuk/modules.js
+  modules/accordion-with-descriptions.js
+  start-modules.js
+)
```

Comment:
I don't think we need to precompile these if we're including the whole tree in application.js with `//= require_tree .` and they're in app/assets.